### ISSUE TITLE
SQL table blob storage

### ISSUE THREAD

**Issue by foxcpp** [foxcpp/maddy#305]
Allows easy backups, etc when used together with storage.imapsql.
See https://github.com/foxcpp/maddy/issues/291#issuecomment-735242195

Depends on #303.

**Comment by theduke** [comment]
Postgres supports [Large Object storage](https://www.postgresql.org/docs/current/lo-interfaces.html) with a custom interface.
That would make this more feasible for non-tiny deployments.

The currently used driver (`pq`) does not support this.
[pgx](https://github.com/jackc/pgx) does though.

**Comment by Avamander** [comment]
> That would make this more feasible for non-tiny deployments.

At first a separate table with a binary data type column would be nice as well. LOS is a bit more involved to keep track of and back up and not everyone might wish to do so.

**Comment by theduke** [comment]
Agreed, for my own deployment a regular table would probably be just fine as well.
